ratis-logservice/src/test/java/org/apache/ratis/logservice/server/TestMetaServer.java

```diff
@@ -126,6 +130,98 @@ public void testReadWritetoLog() throws IOException, InterruptedException {
         assert(res.array().length > 0);
     }
 
+    @Test
+    public void testLogArchival() throws IOException, InterruptedException {
```

Comment:
I really like this test (and `testLogExport`)!

Reply:
Thanks :)